pull-request-process.md

```diff
@@ -1,20 +1,20 @@
-* If a pull request is a typo/grammatical error fix, please make sure that you review the entire document for potential errors. Don’t open separate PRs for small fixes included in the same document.
-* Reference any open issues that your PR can solve, or issues that it is related to.
-* Avoid making large changes in a single commit. Instead, break your request into smaller pieces. This will ensure that reviewers are able to review your PR in a timely manner.
-* Use comments where needed to add additional context
-* Don’t assign an excessive amount of reviewers. If a reviewer is not automatically assigned via the Camunda Community Probot, /assign the pull request to the most relevant project maintainer
-* If your PR is considered a Work-in-Progress [WIP], adding that prefix in your PR title will prevent the PR from being merged until the WIP or /hold command is lifted.
-*  * This can also be achieved by setting your PR as a [`Draft Pull Request`](https://github.blog/2019-02-14-introducing-draft-pull-requests/) on GitHub
-* If your PR has not been reviewed within 14 days, please don’t close your PR and re-open it. Please @ your reviewer and alert a Camunda team member, who will help you triage the pull request if need be.
-* If you feel that your PR isn’t getting the attention it deserves, please post it in the relevant extension’s thread on the [Community Forums](https://forum.camunda.org/c/community-extensions/13) to find additional reviewers.
+1. Follow the directions included in the [pull request template](https://github.com/Camunda-Community-Hub/community/issues/new/choose). This will help the community members and/or extension maintainers who respond to your pull request.
+2. If a pull request is a typo/grammatical error fix, please make sure that you review the entire document for potential errors. Don’t open separate PRs for small fixes included in the same document.
+3. Reference any open issues that your PR can solve, or issues that it is related to.
+4. Avoid making large changes in a single commit. Instead, break your request into smaller pieces. This will ensure that reviewers are able to review your PR in a timely manner.
+5. Use comments where needed to add additional context
+6. Don’t assign an excessive amount of reviewers. If a reviewer is not automatically assigned via the Camunda Community Probot, /assign the pull request to the most relevant project maintainer
+7. If your PR is considered a Work-in-Progress [WIP], adding that prefix in your PR title will prevent the PR from being merged until the WIP or /hold command is lifted.
+    * This can also be achieved by setting your PR as a [`Draft Pull Request`](https://github.blog/2019-02-14-introducing-draft-pull-requests/) on GitHub
+8. If your PR has not been reviewed within 14 days, please don’t close your PR and re-open it. Please @ your reviewer and alert a Camunda team member, who will help you triage the pull request if need be.
```

Comment:
We might need to clarify "@ your reviewer" here in case someone doesn't understand what that means.